rsciio/arina/specifications.yaml

```diff
@@ -0,0 +1,4 @@
+name: arina
+description: Reader for arina 4D-STEM datasets
+extensions: ['.h5']
+non_uniform_axis: false 
```

Comment:
Some keys are missing and this is what make the test suite fails, see the full list in https://hyperspy.org/rosettasciio/contributing.html#defining-new-rosettasciio-plugins.